app/lib/slack.js

```diff
@@ -41,9 +41,16 @@ module.exports = {
       slackOptions.channel = options.channel;
     }
 
-    return new Slack(options.webhookUrl || config.slack.hookUrl, {})
-    .send(slackOptions, (err) => {
-      console.error(err);
+    return new Promise((resolve, reject) => {
```

Comment:
Does this mean we need to update wherever PostMessage is called? I don't know if we always call it as a promise. Like in postActivity()